inc/views/template_parts.php

```diff
@@ -160,8 +167,8 @@ function() use ( $post_id ) {
 			$has_thumbnail_class = apply_filters( 'neve_featured_has_post_thumbnail', '', $post_id );
 			$data                = [
 				'post_id'    => 'post-' . $post_id,
-				'post_class' => $this->post_class( $post_id, 'nv-ft-post ' . $has_thumbnail_class ),
-				'content'    => $this->get_article_inner_content( $post_id ),
+				'post_class' => $this->post_class( $post_id, 'nv-ft-post ' . $has_thumbnail_class, $ordered_components ),
+				'content'    => $this->get_article_inner_content( $ordered_components, $post_id ),
 			];
```

Comment:
If the `$ordered_component` is returned by a function no extra props are needed here.

Reply:
It is not clear. Do you suggest also moving the function for `$ordered_component` call inside those functions? `post_class` has this to maintain compatibility https://github.com/Codeinwp/neve/blob/6ef89cf9251e06b14ee20bbba2ee3cabd6b897c9/inc/views/template_parts.php#L230-L238 , do you want something similar in `get_article_inner_content`?